starknet-foundry/crates/cast/tests/helpers/fixtures.rs

```diff
@@ -175,3 +193,47 @@ pub fn duplicate_directory_with_salt(src_path: String, to_be_salted: &str, salt:
 
     dest_path
 }
+
+pub async fn create_account(salt: &str, add_profile: bool) -> (Utf8PathBuf, &str) {
+    let created_dir = Utf8PathBuf::from(duplicate_directory_with_salt(
+        CONTRACTS_DIR.to_string() + "/v1/balance",
+        "put",
+        salt,
+    ));
+    let accounts_file = "./accounts.json";
+
+    let mut args = vec![
+        "--url",
+        URL,
+        "--network",
+        NETWORK,
+        "--accounts-file",
+        accounts_file,
+        "account",
+        "create",
+        "--name",
+        "my_account",
+    ];
+    if add_profile {
+        args.push("--add-profile");
+    }
+
+    Command::new(cargo_bin!("sncast"))
+        .current_dir(&created_dir)
+        .args(&args)
+        .assert()
+        .success();
```

Comment:
is this a good idea to depend on our bin in fixtures? I'd rather we use `starknet-devnet new_account` and `deploy_account` for this (it also appends to accounts-file)

Reply:
I'm not sure if it appends all the required info (like salt). I can move it from fixtures to `e2e/account/deploy`

Reply:
Yes, it does. I guess it may be a good practice to only use our binary to test things we really want to test - and get rid of multiple points of failures (what if we have a weird bug in account create? deploy tests should probably only test deploy)